docs/guide.md

```diff
@@ -0,0 +1,210 @@
+    src: "./src",
+    assets: "./src/assets",
+    css: {
+      preprocessor: "sass",
+      fileName: "patterns",
+      autoprefixer: {
+        browsers: ["last 2 versions"]
+      }
+    }
+    integrate: "../patternpack-example-app/node_modules/patternpack-example-library",
+    theme: "./node_modules/patternpack-example-theme",
+    logo: "./theme-assets/images/logo.svg",
+    publish: {
+      library: true,
+      patterns: false
+    },
+    patternStructure: [
+      { name: "Atoms", path: "atoms" },
+      { name: "Molecules", path: "molecules" },
+      { name: "Pages", path: "pages" }
+    ],
+    server: {
+      port: 1234
+    }
+  },
+  run: {
+    task: "start"
+  },
+  build: {
+    task: "server"
+  },
+  release: {
+    task: "release"
+  }
+}
+```
+
+There are a few options you might want to customize early on based on your needs:
+
+- `options.css.fileName` - this is the "master" Sass/LESS file that will import your patterns as well as any other dependencies you want to integrate ([documentation](https://github.com/patternpack/patternpack#cssfilename))
+- `options.css.preprocessor` - defaults to Sass, but LESS is available as well ([documentation](https://github.com/patternpack/patternpack#csspreprocessor))
+- `logo` - the default theme populates a placeholder logo, but you can replace it by pointing to an image in your project ([documentation](https://github.com/patternpack/patternpack#logo))
+- `patternStructure` - the taxonomy you will use to organize your Patterns ([documentation](https://github.com/patternpack/patternpack#patternstructure) and [example](https://github.com/patternpack/patternpack#custom-pattern-structure))
+
+## Hello, PatternPack!
+Once you're done configuring PatternPack, you're ready to start it up for the first time. Open a command prompt and you have two options:
+
+```bash
+$ grunt patternpack:run
+# Or if you set up a default grunt task that points at patternpack:run
+$ grunt
+```
+
+This will:
+- Generate the files and folders you'll need
+- Build your Pattern Library in a temporary directory, `./html` or whatever you configured in your [`options.build`](https://github.com/patternpack/patternpack#build).
+- Start up a web server at `http://localhost:8888/`
+- Start up a watch task to auto build any changes
+
+## Adding a Pattern
+*Note that due to limitations in `grunt-watch`, you will need to restart PatternPack every time you add a new file for the watch to recognize it. Once it sees it, all changes with automatically reload.*
+
+Patterns are composed of two files:
+
+- A Markdown (`*.md`) file that contains all the content for a documentation page
+- A Sass/LESS file with the styles for that pattern
+
+To get started, add those two files into one of your Structure folders (if you kept the defaults, it'll be `./src/atoms`, `./src/molecules`, and `./src/templates`).
+
+You can put anything you'd like for the documentation page in the Markdown file, but a good starting point is this template:
```

Comment:
The YAML frontmatter is all that's required - I think you're right about that. I'll articulate that, but besides the YAML, anything else is fair game.